tests/validation/tests/v3_api/test_upgrade.py

```diff
@@ -70,10 +70,13 @@
 app_validate_name = validate_prefix + "-app"
 # the pre_upgrade_externalId is for launching an app
 pre_upgrade_externalId = \
-    "catalog://?catalog=library&template=mysql&version=0.3.7"
+    "catalog://?catalog=test-catalog&template=mysql&version=1.3.1"
```

Comment:
Add a method to create catalog id string from catalog, template, version

Reply:
done. function: create_catalog_external_id in common.py